setup.py

```diff
@@ -74,15 +77,17 @@ def run(self):
 with open('README.rst') as readme:
     long_description = readme.read()
 
-
+extras_require = None
```

Comment:
One blank line needed.

Reply:
Where?

Reply:
Oh, I thought earlier line [76 and 77](https://github.com/coala/coala/pull/4897/files#diff-2eeaed663bd0d25b7e608891384b7298L77) was blank and was good. But after changes, it is not detected as required blank line, then it might be good. Not sure. 😛